services/worker/tests/test_job_manager.py

```diff
@@ -171,24 +192,24 @@ def test_job_runner_set_crashed(
     test_processing_step: ProcessingStep,
```

Comment:
Not part of this PR, but I should have changed the name of this test, maybe only test_set_crashed since the logic is now handled by job_manager and not job_runner.

Reply:
OK... Let's change this when we see it again in a PR